week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,15 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark > 80) {
+    return "A";
+  } else if (80 > mark && mark > 60) {
```

Comment:
This isn't quite correct. Here you're checking that `mark` is greater than 60 **and** it is greater than 80. But we've already checked that `mark` is greater than 80 (in the first `if`). If the computer has got to the `else if`, the `mark` cannot be greater than 80.